database/migrations/2017_04_26_133415_create_trader_user_pivot_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateTraderUserPivotTable extends Migration
```

Comment:
So right now your only users are Service Users?

Who are the entities that log in to use the API?